jscomp/test/record_regression.res

```diff
@@ -100,3 +100,47 @@ let setAA = (ao: option<int>) => {aa: @ns.optional ao, bb: None}
 //     @ns.optional bb: int,
```

Comment:
Pre-existing, but these `@ns.optional` in the file above here are unnecessary since there's proper syntax for that.

Reply:
Removed https://github.com/rescript-lang/rescript-compiler/pull/5827/commits/651ba2b922c9a1406fa4eb985cffc09fd05e98c7